src/App.js

```diff
@@ -4,13 +4,16 @@ import AddTodoForm from './AddTodoForm';
 import TodoList from './TodoList';
 
 function App() {
-  const [newTodo, setNewTodo] = useState(); //use state hook
+  const addTodo = (newTodo) =>{
+    setTodoList([...todoList, newTodo])
+  }
+  
+  const [todoList, setTodoList] = useState([]);
```

Comment:
Nitpick (doesn't require changes):
Typically, you'll find `useState` instantiations located at the top of the component. This way, when they're referenced, like on line 8 above, they're defined before they're used.